src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String color;
+    private double area;
```

Comment:
The `area` should not be a field of the `Figure` class, as the area is specific to each shape and should be calculated in the respective subclass. Instead, you should have an abstract method `calculateArea()` in this class that the subclasses will implement.